package.json

```diff
@@ -21,9 +20,10 @@
     "setup": "npm install && npm run flow-typed",
     "format": "prettier --write --ignore-unknown .",
     "format:check": "prettier --check .",
-    "test": "npm run format:check && npm run lint && npm run flow-typed && npm run flow && npm run jest && npm run karma",
+    "test": "npm run format:check && npm run lint && npm run flow-typed && npm run flow && npm run test:unit",
     "webpack": "babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/webpack --progress",
-    "jest": "jest test/server --env=node --no-cache --collectCoverageFrom='server/' --coverageDirectory='coverage/jest' --coverage --verbose --runInBand --silent=false",
+    "test:unit:watch": "vitest",
+    "test:unit": "vitest run --coverage",
```

Comment:
it would be :100: to standardize w/ the other repos:

paypal checkout components has https://github.com/paypal/paypal-checkout-components/blob/635ab8b162a4ce9a8fb116bdcfc9e4a8edfdd116/package.json#L32C5-L32C31

smart payment buttons has https://github.com/paypal/paypal-smart-payment-buttons/blob/main/package.json#L54 and https://github.com/paypal/paypal-smart-payment-buttons/blob/7b28401d2463c52e7da564e76097690119fde2f7/package.json#L33 but also in vite config https://github.com/paypal/paypal-smart-payment-buttons/blob/main/vite.config.js#L43-L46

not sure what all the differences are but pointing out in case you wanted sdk-client to be the gold standard! 🏅

Reply:
Yeah I LOVE when they all match. Reducing the amount of cognitive load needed for any given repo switch is important to me. 

`vitest run` is needed because vitest defaults to watch mode and `run` disables `watch`. This is used for and needed by CI so it doesn't sit open. To minimize impact to existing paradigms, I went with adding `test:unit:watch`.

`npm test` is a bit overloaded IMO, but didn't want to mess with that in this scope.